### DISCUSSION TITLE
Query regarding the weight option

### DISCUSSION THREAD

**Discussion by JayasimhaNagaraj** [artilleryio/artillery#1095] [Artillery]
Hi Team,

We have 3 APIs in a yaml file which are weighed  : 25,25 and 50 respectively.

During the 1st load test, % of APIs executed were : 20, 28, 52 respectively
During the 2nd load test(repetition of above without any changes) , % of APIs executed were : 28, 19, 53 respectively.

Is there an option to have exactly the same % of APIs across multiple runs.

**Comment by hassy** [comment]
Scenario weights are probabilistic meaning that the exact number of invocations for each scenario will differ slightly between runs: https://artillery.io/docs/guides/guides/test-script-reference.html#Scenario-weights. We don't support fixed percentages for weights at the moment.
